src/datasources/alarms/types/types.ts

```diff
@@ -4,8 +4,13 @@ export interface AlarmsQuery extends DataQuery {
   queryType?: QueryType;
 }
 
+export interface AlarmsVariableQuery extends AlarmsQuery {
+  queryBy?: string;
```

Comment:
```suggestion
  filter?: string;
```

can we use filter for consistency with the alarms query editor types?